java/runtime/src/main/java/io/ray/runtime/gcs/GcsClient.java

```diff
@@ -191,6 +188,18 @@ public JobId nextJobId() {
     return JobId.fromInt(jobCounter);
   }
 
+  public void disconnect() {
+    globalStateAccessor.disconnect();
+  }
+
+  /**
+   * Destroy global state accessor when ray native runtime will be shutdown.
+   */
+  public void destroy() {
+    // Only ray shutdown should call gcs client destroy.
+    globalStateAccessor.destroyGlobalStateAccessor();
```

Comment:
Jut put this in `disconnect`? It's not necessary to have both `disconnect` and `destroy`.

Reply:
Perhaps it's better way to remove disconnect.

> Jut put this in `disconnect`? It's not necessary to have both `disconnect` and `destroy`.